code/modules/admin/verbs/playsound.dm

```diff
@@ -212,6 +221,10 @@
 		SEND_SOUND(M, sound(null))
 		var/client/C = M.client
 		C?.tgui_panel?.stop_music()
+
+	if(S_TIMER_COOLDOWN_TIMELEFT(SStimer, COOLDOWN_INTERNET_SOUND))
+		S_TIMER_COOLDOWN_RESET(SStimer, COOLDOWN_INTERNET_SOUND)
```

Comment:
should this really be a conditional reset or should we just always do this?

Reply:
We may not, I wasn't sure if we'd throw a runtime trying to reset a timer that isn't running. I'll test it.

Reply:
Yeah it's fine to always run.